### DISCUSSION TITLE
Printing Source Code

### DISCUSSION THREAD

**Discussion by prgmr-1** [codefori/vscode-ibmi#2171] [IBM i]
I am old school and occasionally like to print source code,  Don't see that feature or am I just missing something?

**Comment by chrjorgensen** [comment]
@prgmr-1 You can try [this extension](https://marketplace.visualstudio.com/items?itemName=pdconsec.vscode-print), which renders the source as HTML and serves it to your browser for printing.
No syntax colour coding, though...

**Comment by prgmr-1** [comment]
does it imbed in VSCode or is it a separate windows install

**Reply by chrjorgensen** [reply]
When you click on the link, you will see that it's a VS Code extension from the VS Code Marketplace... not a Windows install...
